ci-operator/config/openshift/hypershift/openshift-hypershift-release-4.16__periodics.yaml

```diff
@@ -178,6 +178,7 @@ tests:
   steps:
     cluster_profile: equinix-ocp-metal
     env:
+      ADDITIONAL_NETWORK: "true"
```

Comment:
Since this variable disables usage of the default pod network for the VMs, we're replacing one test path for another.

meaning, this new way of testing will exercise secondary networks as the default network, but as a result we're no longer testing the infra cluster pod network as the default network for the VMs. 

Woudl it make sense for QE to create a separate job to exercise secondary networks? And perhaps only execute that job when verifying a release?